toes/triggers.py

```diff
@@ -43,6 +43,13 @@ async def if_keyword(bot: Client, message: Message, trigger: Trigger, keyword: s
     if keyword in message.content:
         await trigger(bot, message)
 
+@modifier
+async def if_author(bot: Client, message: Message, trigger: Trigger, author_id: str) -> None:
```

Comment:
Any particular reason you chose a string? It'd probably be best to keep it as an integer to match with the other ID fields.

Reply:
fixed